### DISCUSSION TITLE
Subgraph NFTs are missing the "creator", Foundation website does not

### DISCUSSION THREAD

**Discussion by DenisVCode** [f8n/fnd-protocol#227] [Q&A]
Hello!

I am looking around the F8N Mainnet subgraph, however, in the queries I have noticed that there is often creator stated as null. 
(example NFT id: 0x002ed370642aad1352b543ab0a71721fc5916d07-1, shows with owner: null in graph, however in FND creator is correctly stated https://foundation.app/@mitji/trblnxc/1)

By my approximation, this is about 20% of all NFTs. Considering that Foundation website shows the data correctly, what piece of the puzzle am I missing?

Happy to supply more examples if needed. 

Thanks!

**Comment by HardlyDifficult** [comment]
Our subgraph only fully supports NFTs created on Foundation. The example you provided was created on Manifold - for those NFTs you would need to hit the contract directly to get the creator, or potentially the Manifold subgraph if they have one. The reason for this is creator is a non-standard feature and different teams may have approached this differently than how our contracts emit the information.

hth
